ui/Screen/Onboarding.svelte

```diff
@@ -38,9 +40,11 @@
 
   const onCreateIdentity = async (handle, passphrase) => {
     try {
+      screen.lock();
+      requesting = true;
+      notification.info("Creating the identity...");
```

Comment:
Can we remove the notification?
It feels to me like it's just adding noise because the identity creation in production builds is fairly fast.

![prod-build](https://user-images.githubusercontent.com/158411/98253750-bc912780-1f7b-11eb-9a07-a296f1612009.gif)